view/template/page/press-kit.php

```diff
@@ -42,7 +42,8 @@
           <h4>
             <?php echo $metadata['name'] ?>
             <?php if (isset($metadata['email'])): ?>
-              <a href="mailto:<?php echo $metadata['email'] ?>" class="link-primary"><?php echo $metadata['email'] ?></a>
+              <a href="mailto:<?php echo $metadata['email'] ?>" class="link-primary"><span class="icon icon-envelope"></span></a><a href="https://github.com/<?php echo $metadata['github'] ?>" class="link-primary"><span class="icon icon-github"></span></a>
```

Comment:
One little problem with this code, currently if a person has no twitter the twitter icon appears.  The proper behaviour should be that there is no twitter icon if the user has none. The solutions would be to check if the ```$metadata['twitter']``` is set.

Reply:
@maximest-pierre it just for founding team. and some people with role founding team have twitter id  . only 4 people marked as founding team. you'd need check on lbry.io/team .

Reply:
jack doesn't have a twitter account.